catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/configurators/KarafConfigurator.java

```diff
@@ -42,7 +43,8 @@ public static Option karafConfiguration() {
             .unpackDirectory(new File("target", "exam"))
             .useDeployFolder(false),
         configureConsole().ignoreLocalConsole(),
-        logLevel().logLevel(LogLevelOption.LogLevel.WARN),
+        keepRuntimeFolder(),
```

Comment:
Doesn't need to be dealt with now, but it would be worth thinking about how we're going to approach `-DkeepRuntimeFolder` and what exactly that means for the build now that we have multiple places where karaf containers are spun up

Should the one property cause all karaf containers to be kept? Should we have properties for each component test? Should it only affect the itests?